Invati Service Manager - New Actions - Create Incident, Update Incident, Delete Incident and Add Note (#636)
rapid7/insightconnect-plugins: ivanti_service_manager/help.md

# Description

> [Ivanti Service Manager](https://www.ivanti.co.uk/products/service-manager) is a flexible and complete cloud-optimized, easily scalable and adaptable ITSM solution used for managing and automating ITSM processes

Reviewer: ```suggestion
[Ivanti Service Manager](https://www.ivanti.co.uk/products/service-manager) is a flexible and complete cloud-optimized, easily scalable and adaptable ITSM solution used for managing and automating ITSM processes.
```

# Key Features

* Create incidents
* Update incidents
* Delete incidents
* Add note

# Requirements

> * Requires an API Key from the product

Reviewer: ```suggestion
* Requires an API Key from Ivanti
```

# Documentation
## Setup

> Follow below steps in order to generete an API Key in Ivanti Service Manager to use in this plugin:

Reviewer: ```suggestion
Follow the steps below to generate an API Key in Ivanti Service Manager to use with this plugin:
```

1. From the Configuration console, click Configure > Security Controls > API Keys
2. Select the relevant group created for REST API from the Key Groups section. 
3. Click Add API Key. The application displays the New API Key page.

> 4. Give the key an appropriate permissions to perform the action.

Reviewer: ```suggestion
4. Give the key the appropriate permissions to perform the action.
```

5. Click Save Key. The generated REST API Key is saved with the details you entered.

For more detailed instruction please refer to [Ivanti Service Manager API Guide](https://help.ivanti.com/ht/help/en_US/ISM/2020/admin/Content/Configure/API/Using-REST-API-Key.htm)

The connection configuration accepts the following parameters:

|Name|Type|Default|Required|Description|Enum|Example|
|----|----|-------|--------|-----------|----|-------|
|credentials|credential_secret_key|None|True|API key from account|None|1A2B3CC4D5E67F8901G2HI345J6K7L89|
|ssl_verify|boolean|True|False|Validate TLS / SSL certificate|None|True|
|url|string|None|True|API access URL|None|https://example.com|

Example input:

```
{
  "credentials": "1A2B3CC4D5E67F8901G2HI345J6K7L89",
  "ssl_verify": true,

>   "url": "https://example.com"

Reviewer 2: Checking with Ivanti on the best URL example.
